conf/env/PRESTASHOP.env

```diff
@@ -16,6 +16,7 @@ PS_FOLDER_INSTALL=installOLD
 ADMIN_MAIL=demo@reverb.com
 ADMIN_PASSWD=123456
 PS_HANDLE_DYNAMIC_DOMAIN=0
+PS_ERASE_DB=1
```

Comment:
It's a new variable to re-install the database. If the value is 0, when we launch launch the Docker container, the system don't install the database.